src/test/java/com/jcabi/ssh/SSHTest.java

```diff
@@ -78,13 +88,12 @@ public void executeCommandOnServer() throws Exception {
             .usePublicKeyAuthentication().build();
         sshd.setCommandFactory(new MkCommandCreator());
         sshd.start();
-        final String cmd = "some test command";
+        final String cmd = SOME_TEST_COMMAND;
```

Comment:
@LinkedList please, inline this string back and use different strings in each file.
Usually, we are trying to avoid coupling between tests